### PR TITLE
copied HTML from my-portfolio

### PR DESCRIPTION
**Pull request by Ta-shed**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Website</title>
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+</head>
+<body>
+<nav class="bg-indigo-800 flex flex-row text-white h-1/6 justify-center space-x-6 ">
+    <a class="hover:text-gray-900 shadow" href="#">Home</a>
+    <a class="hover:text-gray-900" href="contact.html">Contact</a>
```
Comment (luciavelasco) [inline index.html:11]: Change the `href` to `#` (unless you have a contact.html file that you're writing)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <a class="hover:text-gray-900" href="blog.html">Blog</a>
```
Comment (luciavelasco) [inline index.html:12]: Change the `href` to `#` (unless you have a blog.html file that you're writing)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    </nav>
+<main class=" h-screen  bg-black flex flex-row">
+    <section class="h-auto w-11/12 m-8 bg-yellow-300 flex flex-row
+    justify-evenly justify-center items-end">
```
Comment (luciavelasco) [inline index.html:16]: you're using both `justify-evenly` and `justify-center`. One of them is overriding the other, decide which one you want and remove the other one

**Review by luciavelasco — COMMENTED** [review]
Good work! Good use of Tailwind.

File: index.html (hunk 1 above)
Comment (Ta-shed) [reply index.html:11]: thanks. I have decided to  replace with href="mailto:tzvekare@gmail.com? subject= Hello Tariro"> as per your illustration earlier today

File: index.html (hunk 2 above)
Comment (Ta-shed) [reply index.html:12]: Agree. thank you.

File: index.html (hunk 3 above)
Comment (Ta-shed) [reply index.html:16]: I will keep justify-evenly thanks

**Pull request merged**